### DISCUSSION TITLE
MHI2_ER_SEG11_P4517_MU1219

### DISCUSSION THREAD

**Discussion by nicedevil007** [Mr-MIBonk/M.I.B._More-Incredible-Bash#122] [Q&A]
Hi there, I have a relativly old MIB2High in my loved Seat Leon 5F. Hardware version H41.
I go the livetime maps enabled by a friend about 3-5 years ago (can't remember).

Then an other friend told me from this gitrepo. I decided to take a look at it and copied all the files to a new SD card, plugged it in and went the "autoinstall" route with `Swdlautorun.txt`.

After everything was done I'm still on P4517_MU1219, so I guess I missed a step. I hope someone can help me out?

At least in the wiki I found those versions that should be able to be installed right?
![image](https://user-images.githubusercontent.com/17103076/220363680-544ee33b-77f6-46b7-bc6c-6c6c72af51eb.png)

**Comment by harman-f** [comment]
This repo does not hold FW Updates. 
M.I.B is a tool to patch and configure MHI2 units. 

You want to look for the AIO FW Update which is around in many places.

**Comment by nicedevil007** [comment]
Ahhh so here is the missunderstanding on my side :)
So an other friend got the same model (Mib2High, just later Hardware) and got the error with navigation database is not activated.
This is doable with M.I.B. ? (SWD fix?)

**Reply by harman-f** [reply]
https://www.seatcupra.net/forums/threads/mib2-mhi2-harman-high-aio-fw-update-ifs-root-patch-asi-carplay-android-auto-map-update.469828/#post-5012792
